<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="36456" started-at="2019-03-23T05:38:25Z" finished-at="2019-03-23T05:39:01Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="36456" started-at="2019-03-23T05:38:25Z" finished-at="2019-03-23T05:39:01Z">
      <class name="Tests.FlightFinderTest">
        <test-method status="PASS" signature="setupEnvironment()[pri:0, instance:Tests.FlightFinderTest@735b5592]" name="setupEnvironment" is-config="true" duration-ms="27576" started-at="2019-03-23T05:38:25Z" finished-at="2019-03-23T05:38:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupEnvironment -->
        <test-method status="PASS" signature="verifyRadioSize()[pri:0, instance:Tests.FlightFinderTest@735b5592]" name="verifyRadioSize" duration-ms="7673" started-at="2019-03-23T05:38:52Z" description="Verify size of Radio" finished-at="2019-03-23T05:39:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRadioSize -->
        <test-method status="PASS" signature="verifyNewYorkIsSelected()[pri:1, instance:Tests.FlightFinderTest@735b5592]" name="verifyNewYorkIsSelected" duration-ms="158" started-at="2019-03-23T05:39:00Z" description="Verify New York is Selected in From Port" finished-at="2019-03-23T05:39:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyNewYorkIsSelected -->
        <test-method status="PASS" signature="verifyBusinessClassIsSelected()[pri:2, instance:Tests.FlightFinderTest@735b5592]" name="verifyBusinessClassIsSelected" duration-ms="484" started-at="2019-03-23T05:39:00Z" description="Verify Business Class is selected in Serve Class" finished-at="2019-03-23T05:39:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyBusinessClassIsSelected -->
        <test-method status="FAIL" signature="verifyDepartFromListVal()[pri:3, instance:Tests.FlightFinderTest@735b5592]" name="verifyDepartFromListVal" duration-ms="158" started-at="2019-03-23T05:39:01Z" finished-at="2019-03-23T05:39:01Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[verify Drop down List: lists don't have the same size expected [10] but found [1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: verify Drop down List: lists don't have the same size expected [10] but found [1]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:819)
at Tests.FlightFinderTest.verifyDepartFromListVal(FlightFinderTest.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyDepartFromListVal -->
        <test-method status="PASS" signature="closeEnvironment()[pri:0, instance:Tests.FlightFinderTest@735b5592]" name="closeEnvironment" is-config="true" duration-ms="94" started-at="2019-03-23T05:39:01Z" finished-at="2019-03-23T05:39:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeEnvironment -->
      </class> <!-- Tests.FlightFinderTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
